build.gradle.kts

```diff
@@ -20,6 +20,13 @@ version = properties("pluginVersion")
 // Configure project's dependencies
 repositories {
     mavenCentral()
+    maven("https://packages.jetbrains.team/maven/p/ij/intellij-dependencies") // TODO: Remove when published to MavenCentral
+}
+
+dependencies {
+    testImplementation("com.intellij.remoterobot:remote-robot:0.11.6")
```

Comment:
externalize versions

Reply:
There's no need to – Dependabot will take care of it anyway.

Reply:
one might want to set specific version in case of regressions or incompatibility with chosen target IDEs